saleor/graphql/order/mutations/orders.py

```diff
@@ -538,6 +561,15 @@ def get_instance(cls, info, **data):
                     )
                 }
             )
+        if not instance.lines.count():
```

Comment:
`exists` should be faster when we only want to check the existence of an object.

```suggestion
        if not instance.lines.exists():
```